ipatests/test_integration/test_ntp_options.py

```diff
@@ -106,89 +95,85 @@ def test_server_replica_client_install_with_pool_and_srv(self):
         test to verify that ipa-server, ipa-replica and ipa-client install
         passes with options --ntp-pool and --ntp-server together
         """
-        ntp_pool = "pool.ntp.org"
-        ntp_server = "1.pool.ntp.org"
         expected_msg = "Configuration of chrony was changed by installer."
-
-        server_install = tasks.install_master(
-            self.master, setup_dns=False,
-            extra_args=['--ntp-pool=%s' % ntp_pool,
-                        '--ntp-server=%s' % ntp_server])
+        args = ['--ntp-pool=%s' % self.ntp_pool,
+                '--ntp-server=%s' % self.ntp_server1]
+        server_install = tasks.install_master(self.master, setup_dns=False,
+                                              extra_args=args)
         assert expected_msg in server_install.stderr_text
         cmd = self.master.run_command(['cat', paths.CHRONY_CONF])
-        assert ntp_pool in cmd.stdout_text
-        assert ntp_server in cmd.stdout_text
+        assert self.ntp_pool in cmd.stdout_text
+        assert self.ntp_server1 in cmd.stdout_text
 
-        replica_install = self.install_replica('--ntp-pool=%s' % ntp_pool,
-                                               '--ntp-server=%s' % ntp_server)
+        replica_install = tasks.install_replica(self.master, self.client,
```

Comment:
@Tiboris Here we need one step replica install to test ntp options.
but in this case, it is trying to use Replica promotion with ntp options.

Reply:
OK